src/org/usfirst/frc4904/robot/humaninterface/operators/DefaultOperator.java

```diff
@@ -0,0 +1,23 @@
+package org.usfirst.frc4904.robot.humaninterface.operators;
+
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.robot.commands.IntakeSquared;
+import org.usfirst.frc4904.robot.commands.OuttakeSquared;
+import org.usfirst.frc4904.standard.humaninput.Operator;
+
+public class DefaultOperator extends Operator {
+	public DefaultOperator() {
+		super("DefaultOperator");
+	}
+
+	public DefaultOperator(String name) {
+		super(name);
+	}
+
+	@Override
+	public void bindCommands() {
+		RobotMap.Component.joystick.button3.onlyWhileHeld(new IntakeSquared());
+		RobotMap.Component.joystick.button4.onlyWhileHeld(new OuttakeSquared());
```

Comment:
we need buttons 5 & 6 to be mapped to forcefully opening and closing the rollyBOI pistons, in that order.